tsdb/head.go

```diff
@@ -340,7 +340,7 @@ type headMetrics struct {
 	chunksRemoved             prometheus.Counter
 	gcDuration                prometheus.Summary
 	samplesAppended           *prometheus.CounterVec
-	outOfOrderSamplesAppended prometheus.Counter
+	outOfOrderSamplesAppended *prometheus.CounterVec
```

Comment:
this was already updated to a countervec in https://github.com/prometheus/prometheus/commit/52389647b2ab4e52d86c97089cf5f4260839f984